bin/resources/GameIndex.yaml

```diff
@@ -41918,9 +41923,14 @@ SLUS-20857:
   region: "NTSC-U"
   compat: 5
 SLUS-20858:
-  name: "Corvette 50th Anniversary"
```

Comment:
Isn't this correct?

Reply:
Not according to redump it's simply called "Corvette".

http://redump.org/disc/12037/